src/components/CardExpirationDate/CardExpirationDate.jsx

```diff
@@ -1,9 +1,16 @@
-import PropTypes from 'prop-types';
+import { useContext } from 'react';
+import { CardInfoContext } from 'CardInfoContextProvider';
 
-import { MONTH, LIMIT_LENGTH } from 'constants';
+import { MONTH, LIMIT_LENGTH, ACTION } from 'constants';
 import { inputNumberOnly, limitInputLength } from 'utils';
 
-function CardExpirationDate({ cardExpirationDate, setCardExpirationDate }) {
+function CardExpirationDate() {
+  const { state, dispatch } = useContext(CardInfoContext);
+
+  const cardExpirationDate = { month: state.card.month, year: state.card.year };
+  const setCardExpirationDate = (cardExpirationDate) =>
+    dispatch({ type: ACTION.SET_CRAD_EXPIRATION_DATE, cardExpirationDate });
```

Comment:
`CardInfoContext` 를 사용하는 훅을 하나 만들어 아래와 같이 개선해볼 수 있을까요? `cardExpirationDate`, `setCardExpirationDate` 을 내보내는 건 Provider 에서 제공해주어도 될 것 같고, 훅에서 만들어 제공해주어도 될 것 같습니다;

```suggestion
  const { cardExpirationDate, setCardExpirationDate } = useCardInfoContext()
```